index.d.ts

```diff
@@ -60,22 +61,13 @@ declare namespace LightMyRequest {
     cookies?: { [k: string]: string }
   }
 
-  interface Request extends stream.Readable {
-    url: string
-    httpVersion: string
-    method: HTTPMethods
-    headers: http.IncomingHttpHeaders
-    rawHeaders: string[]
-    prepare: (next: () => void) => void
-    // @deprecated
-    connection: object
-  }
+  interface Request extends http.IncomingMessage {}
```

Comment:
Are there any properties that we consider public that aren't part of the plain `http` objects?

Reply:
I would just drop this interface.